rpc/wrpc/wasm/src/client.rs

```diff
@@ -957,6 +945,8 @@ build_wrpc_wasm_bindgen_interface!(
         /// performance and status of the Kaspa node.
         /// Returned information: Memory usage, CPU usage, network activity.
         GetMetrics,
+        /// Retrieves current number of network connections
+        GetConnections,
```

Comment:
Should `GetSystemInfo` be added somewhere here too?

Reply:
I suppose.  This was added specifically for `kaspa-resolver` machine profiling, which is why it was missed.